models/src/main/java/com/vimeo/networking2/TextTrack.kt

```diff
@@ -1,58 +1,57 @@
 package com.vimeo.networking2
 
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
 import com.vimeo.networking2.enums.TextTrackType
-import java.util.*
 
 /**
  * Information on a text text.
  */
+@JsonClass(generateAdapter = true)
 data class TextTrack(
 
     /**
      * Whether this text track is active.
      */
+    @Json(name = "active")
     val active: Boolean? = null,
 
     /**
      * The read-only URL of the text track file, intended for use with HLS playback.
      */
+    @Json(name = "hsl_link")
     val hlsLink: String? = null,
 
-    /**
-     * The time in ISO 8601 format when the read-only HLS playback text track file expires.
-     */
-    val hlsLinkExpiresTime: Date? = null,
```

Comment:
looks like there was two expiration properties that have been removed, intentional?

Reply:
I added back in.